examples/examples/toggle-layers.rs

```diff
@@ -0,0 +1,47 @@
+#![deny(rust_2018_idioms)]
```

Comment:
Can we add this to `examples/README.md`, which has a list of all the examples and what they demonstrate?